README.md

```diff
@@ -1,33 +1,81 @@
+## When to Follow the Process
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+Consider employing this process if you intend to make substantial changes to the Bicep programming language that would affect user experience. Example where the use of a REP is warranted include:
+- *Introduction of New Features*: If the proposed change involves the addition of a new syntax kind or a new ARM template function, particularly if it necessitates the use of a feature flag.
+- *Breaking Changes to Non-Experimental Features*: Any alterations that may potentially disrupt existing functionality of non-experimental features.
+- *Feature Deprecation*: When contemplating the replacement of a feature that has already been shipped with a new feature.
 
-## Contributing
+It's important to note that numerous other changes, such as bug fixes and minor feature enhancements, can be implemented through the regular issue triage and pull request review process within the primary Bicep repository.
 
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+## Who Should Create Bicep REPs
 
-When you submit a pull request, a CLA bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
+While members of the Bicep community are allowed to contribute by authoring new REPs and providing feedback, it's important to note that, in practice, Bicep REPs are typically submitted by the Bicep core maintainers. This submission follows an exclusive design and discussion process among the maintainers. The author of a Bicep REP is not only responsible for the initial design but also for implementing the proposed feature and overseeing the REP's lifecycle.
 
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+Therefore, it is generally not recommended for community members to independently initiate the creation of REPs. Instead, we encourage active collaboration and engagement within the main Bicep repository. This involves participating in issue discussions and upvoting feature requests, which can prompt the Bicep core maintainers to initiate the creation of Bicep REPs.
```

Comment:
should we be more specific thatm in most cases, starting with an issue for a feature proposal is more appropriate?

Reply:
Yeah makes sense. Incorporated that into this section.